var/spack/repos/builtin/packages/rocblas/package.py

```diff
@@ -133,6 +133,7 @@ def check(self):
     patch('0002-Fix-rocblas-clients-blas.patch', when='@4.2.0:4.3.1')
     patch('0003-Fix-rocblas-gentest.patch', when='@4.2.0:5.1')
     patch('0004-Find-python.patch', when='@master:')
+    patch('0004-Find-python.patch', when='@5.2.0:')
```

Comment:
You can delete

    patch('0004-Find-python.patch', when='@master:')

Reply:
Deleted patch('0004-Find-python.patch', when='@master:')